package-lock.json

```diff
@@ -47468,7 +47476,7 @@
         "colorthief": "^2.3.2",
         "draft-js": "^0.11.7",
         "draft-js-export-html": "^1.4.1",
-        "draft-js-import-element": "git+ssh://git@github.com/swissspidy/draft-js-import-element.git",
+        "draft-js-import-element": "https://github.com/swissspidy/draft-js-import-element",
```

Comment:
Hmm, are these changes intended? Do they matter? Ping @swissspidy

Reply:
That usually happens when you configure Git to pull via HTTPS and push via SSH (and someone else hasn't configured it like that). Then a change like this to the lock file can happen. Doesn't really cause issues besides unnecessarily changing files.